polls/tool/src/bundle/org/sakaiproject/poll/bundle/Messages.properties

```diff
@@ -61,6 +63,7 @@ warn_poll_has_votes=You are editing a poll that has already been voted on
 new_poll_noperms=you do not have permission to add new polls in this site
 
 new_option_title=Add an Option
+new_option_batch_title=Add options massively
```

Comment:
See my comment above about this wording.

Reply:
makes sense to me, thanks